apm-sniffer/apm-sdk-plugin/mysql-6.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/jdbc/mysql/v6/define/FailoverConnectionProxyInstrumentation.java

```diff
@@ -54,4 +53,9 @@
     protected ClassMatch enhanceClass() {
         return byName(INTERCEPT_CLASS);
     }
+
+    @Override
+    protected String[] witnessClasses() {
```

Comment:
I advise that this method should remove. because this method had implemented in its parent class